src/components/TranslationPanel.js

```diff
@@ -38,7 +38,12 @@ class TranslationPanel extends React.Component {
 			$('div[data-verse="r' + (limits[0] + 1) + '"]').css({ "border-radius": "10px 10px 0px 0px" });
 			$('div[data-verse="r' + (limits[1] + 1) + '"]').css({ "border-radius": "0px 0px 10px 10px" });
 		}
-		}*/}
+        }*/}
+        document.getElementById(vId).addEventListener("paste", function (e) {
+			e.preventDefault();
+			var text = e.clipboardData.getData("text/plain");
+			document.execCommand("insertHTML", false, text);
+		})
```

Comment:
So this function will fire whenever we attempt to paste in the translation panel, right?

Reply:
yes exactly!

Reply:
there are some other ways also. By getting to know if there are some child nodes contained on the pasted element and get the inner element....but I find this method is better, as it manages with clipboard data.